dojo/templates/base.html

```diff
@@ -556,6 +556,13 @@
                                                         {% trans "Vulnerability Black & White Lists" %}
                                                     </a>
                                                 </li>
+                                                {% if request.user|is_in_reviewer_group %}
```

Comment:
register new function has_permission_to_reclassify_orphans in dojo/templatetags/authorization_tags.py